src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -313,12 +314,20 @@ protected internal override void Write(int pinNumber, PinValue value)
             *registerPointer = register;
         }
 
-        public void GetRegisters(out ulong* setRegister, out ulong* clearRegister)
+        protected virtual ulong SetRegister
         {
-            Initialize();
+            get { return *(ulong*)(_registerViewPointer->GPSET); }
+
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
only setter needs it and not getter? 😄

Reply:
I added to the setter and also I removed the virtual specifier to ensure the in-lining will occur.